generator-prisma-client/nuxt3-starter-nodejs/package.json

```
@@ -10,20 +10,18 @@
     "postinstall": "nuxt prepare"
   },
   "dependencies": {
-    "@prisma/adapter-pg": "6.11.0",
-    "@prisma/client": "6.11.0",
+    "@prisma/adapter-pg": "6.14.0",
+    "@prisma/client": "6.14.0",
     "nuxt": "3.17.6",
     "vue": "3.5.17",
     "vue-router": "4.5.1"
   },
   "devDependencies": {
     "@nuxtjs/tailwindcss": "6.14.0",
-    "prisma": "6.11.0",
+    "dotenv": "^17.2.1",
```

```suggestion
    "dotenv": "17.2.1",
```

At least all other dependencies are using a fixed version.